docker/README.md

```diff
@@ -0,0 +1,87 @@
+# Docker's version of `amc2moodle`
+
+A docker version of `amc2moodle` is available and it starts a daemon server to automatically execute `amc2moodle` or `moodle2amc`.
+
+Alternatively, it is also possible to run an interactive shell in the docker to recover standard `amc2moodle` command line interface.
+
+# Installation
+
+The container can be downloaded using the following command 
+```
+docker pull ghcr.io/luclaurent/amc2moodle:latest
```

Comment:
Perhaps we should convert all  `ghcr.io/luclaurent` to `ghcr.io/nennigb` for concistency

Reply:
Yes